cdc/model/sink.go

```diff
@@ -377,10 +377,11 @@ func (r *RowChangedEvent) ApproximateBytes() int {
 
 // Column represents a column value in row changed event
 type Column struct {
-	Name  string         `json:"name" msg:"name"`
-	Type  byte           `json:"type" msg:"type"`
-	Flag  ColumnFlagType `json:"flag" msg:"-"`
-	Value interface{}    `json:"value" msg:"value"`
+	Name    string         `json:"name" msg:"name"`
+	Type    byte           `json:"type" msg:"type"`
+	Charset string         `json:"-"`
```

Comment:
Do we need charset to redo changes to downstream MySQL? @ben1009

Reply:
if the info is need when write to downstream through sink then yes, it is needed when rewrite the event